client/my-sites/plans/current-plan/index.jsx

```diff
@@ -33,14 +33,14 @@ class CurrentPlan extends Component {
 	static propTypes = {
 		selectedSiteId: PropTypes.number,
 		selectedSite: PropTypes.object,
-		sitePlans: PropTypes.object,
+		isRequestingSitePlans: PropTypes.bool,
 		context: PropTypes.object
 	};
 
 	isLoading() {
-		const { selectedSite, sitePlans } = this.props;
+		const { selectedSite, isRequestingSitePlans: isRequestingPlans } = this.props;
 
-		return ! selectedSite || ! sitePlans.hasLoadedFromServer;
+		return ! selectedSite || isRequestingPlans;
```

Comment:
Well, as far as I know, we always show placeholders when data is being loaded.

So, if I understood you correctly, you want to show placeholders only when there is no selected site (which should not happen as user is presented with site picker if no site is selected). and otherwise, show persisted data (if there are any)?

I'm for showing persisted data if it's available but for that, I don't think checking just for `! selectedSite` in `isLoading` is enough due to the reason mentioned above. How would we know whether we have some persisted data? Just by looking into variables such as `currentPlan` (whether they contain something)?